src/Test/Perf/helloworld/HelloWorld.csx

```diff
@@ -0,0 +1,20 @@
+#load "../util.csx"
+using System.IO;
+
+var pathToCsc = Path.Combine(BinDebugDirectory(), "csc.exe");
+var pathToHelloWorld = Path.Combine(MyWorkingDirectory(), "HelloWorld.cs");
+var pathToOutput = Path.Combine(MyArtifactsDirectory(), "HelloWorld.exe");
+
+ProcessResult result;
+
+var msToCompile = WalltimeMs(out result,
```

Comment:
This is fine as an initial starting point, but we should be profiling (with PerfView/EventTrace, etc) and pulling the metrics from there.  I don't want to have tests where we don't have profiles to analyze.  The pattern I would expect is:
```
StartProfiler();
ShellOut(...);
StopProfiler();
Report();
```
Report will either require a file on the side that describes all the processes/metrics we're interested in reporting, or we can find a way to encode that in the test and pass it as an argument.